README.md

```
@@ -81,6 +81,29 @@ or
 
     mvn clean verify -P !fed-build -DskipTests &&mvn -P cargo.run
 
+## Validating Changes in an Integrated Environment
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)